navigation-timing/nested_unload_timing.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="timeout" content="long">
+    <title>Navigation Timing: unload event with nested contexts</title>
+    <link rel="help" href="https://w3c.github.io/navigation-timing/"/>
+    <script src="/common/utils.js"></script>
+    <script src="/common/dispatcher/dispatcher.js"></script>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+
+    const dummyURL = () => `/navigation-timing/resources/blank_page_green.html?uid=${token()}`;
+
+    promise_test(async t => {
+        const mainWindowUnloadDuration = 100;
+        const iframeUnloadDuration = 400;
+        /*
+            We create 3 contexts: one for the document to unload, one for the iframe in that document,
+            and one for the document that's loaded after the unload.
```

Comment:
If I may suggest some slight clarifications: "One for a popup window that will load a document and then later unload it, one for an iframe inside that popup window, that will unload with the parent document as well, and one for document the popup window will load in order to trigger those previous unload events"